LayoutTests/platform/gtk-wk2/svg/animations/animate-zero-keyPoints-should-not-crash-expected.txt

```diff
@@ -0,0 +1,15 @@
+layer at (0,0) size 800x600
```

Comment:
This is a crashing bug. We do not need to dump the layout tree in this case. All we need is to be able to print a text message saying "We did not crash". We should only have one text expectation file.